datayoga-py/README.md

```diff
@@ -124,14 +124,18 @@ DataYoga supports both SQL and [JMESPath](https://jmespath.org/) expressions. JM
+| `mid `               | Returns a specified number of characters from the middle of a given text string                                                  | Input: `{"greeting": "hello world!"}` <br /> Expression: `` mid(greeting, `4`, `3`) `` <br /> Output: `o w`                                       |
+| `replace`            | Replaces all the occurrences of a substring with a new one                                                                       | Input: `{"sentence": "one four three four!"}` <br /> Expression: `replace(sentence, 'four', 'two')` <br /> Output: `one two three two!`           |
+| `right`              | Returns a specified number of characters from the end of a given text string                                                     | Input: `{"greeting": "hello world!"}` <br /> Expression: `` right(greeting, `6`) `` <br /> Output: `world!`                                       |
+| `split`              | Splits a string into a list of strings after breaking the given string by the specified delimiter (comma by default)             | Input: `{"departments": "finance,hr,r&d"}` <br /> Expression: `split(departments)` <br /> Output: `['finance', 'hr', 'r&d']`                      | Default delimiter is comma - a different delimiter can be passed to the function as the second argument, for example: `split(departments, ';')` |
+| `upper`              | Converts all lowercase characters in a string into uppercase characters                                                          | Input: `{"fname": "john"}` <br /> Expression: `upper(fname)` <br /> Output: `JOHN`                                                                |
+| `uuid`               | Generates random uuid4 and returns it as string in standard format.                                                              | Input: None <br /> Expression: `uuid()` <br /> Output: `3264b35c-ff5d-44a8-8bc7-9be409dac2b7`                                                     |
+| `hash`               | Calculates hash using given `hash_name` hash function and returns hexadecimal representation.                                    | Input: `{"some_str": "some_value"}` <br /> Expression: `hash(some_str, &#96;sha1&#96;)` <br /> Output: `8c818171573b03feeae08b0b4ffeb6999e3afc05` | Supported hash functions: sha256, md5, sha384, sha3_384, blake2b, sha1, sha512, sha3_224, sha224, sha3_256, sha3_512, blake2s
+| `time_delta_days`    | Returns the number of days between given `dt` and now(positive) or the number of days that have passed from now(negative).       | Input: `{"dt": '2021-10-06T18:56:16.701670+00:00'}` <br /> Expression: `time_delta_days(dt)` <br /> Output: `365`                                 | If `dt` is string ISO datetime(2011-11-04T00:05:23+04:00) is assumed. If `dt` is number unix timestamp is assumed.
```

Comment:
If `dt` is a string,  ISO datetime (2011-11-04T00:05:23+04:00, for example) is assumed

Reply:
✅